src/lib/browse/layers/PointAmenityLayerControl.svelte

```diff
@@ -12,23 +12,27 @@
   // - The layer name, for layerZorder
   // - A color name in colors.ts
   export let name: string;
-  // Uncapitalized
-  export let singularNoun: string;
   // Capitalized
   export let pluralNoun: string;
 
   export let circleRadius: number;
 
+  export let url: string;
+
+  export let tooltip: (feature: MapGeoJSONFeature) => string;
+
+  const originalOnClick = (e: CustomEvent<MapGeoJSONFeature>) => {};
+
+  export let onClick = originalOnClick; 
+
+  $: clickable = onClick !== originalOnClick;
```

Comment:
This is a bit funky I'll admit, might be a better way to do

Reply:
My preference is to not have this indirection at all (see above). But if we keep it, this doesn't need to be reactive, because `InteractiveLayer` and such don't expect things like this to change after creation. It could just be a regular `let clickable = ....` calculated once.